chat/input-bar/src/InputBar/InputBar.tsx

```diff
@@ -366,6 +368,12 @@ export const InputBar = forwardRef<HTMLFormElement, InputBarProps>(
         return;
       }
 
+      if (isLoading && onClickStop) {
```

Comment:
why would the submit handler be fired if `isLoading` is true? Shouldn't those be separate events?